scripts/DCOSWindowsAgentSetup.ps1

```diff
@@ -332,7 +333,7 @@ Write-Log "run = "+$mesos_run
     Invoke-Expression ($global:NssmDir+"\nssm set Mesos Type SERVICE_INTERACTIVE_PROCESS" )
     Invoke-Expression ($global:NssmDir+"\nssm set Mesos AppThrottle 1500" )
     Invoke-Expression ($global:NssmDir+"\nssm set Mesos AppStdout "+$global:MesosLogDir+"\MesosStdOut.log" )
-    Invoke-Expression ($global:NssmDir+"\nssm set Mesos AppStderr "+$global:MesosLogDir+"\MesosStdErr.log" )
+    Invoke-Expression ($global:NssmDir+"\nssm set Mesos AppStderr "+$global:MesosLogDir+"\mesos-agent.log" )

```

Comment:
stdout is not changed from MesosStdOut.log, but stderr is changed to mesos-agent.log? Don't both need to be changed to the same one? I am assuming the --external_log_file makes both of these irrelevant anyway, right?

Reply:
I have the same concern as Paul.

Reply:
I talked with Paul about this back about a month ago, mesos-agent does not log its agent log messages to stdout, only to stderr. On any system I've seen, I've yet to see MesosStdOut.log have anything inside of it. I've seen the same on Linux systems.